static/flow/views/data-set-view.js

```diff
@@ -483,11 +463,11 @@ var DataSetView = function(options) {
         if (base.m_plotHandler.intervalSelect) {
             base.m_plotHandler.setIntervalSelect(false);
             $('#data-set-select-interval').html('Select Interval');
-            $('#data-set-select-interval').removeClass('btn-info');
+            //$('#data-set-select-interval').removeClass('btn-info');
```

Comment:
No, probably just an oversight.  The few places where commented code is left in (at least when I leave it in) is because it is a potential functional path that we might end up using.

Reply:
I'd still probably still delete it since it's all in source control. Otherwise, I think it's more likely to create bloat and it's never clear when to delete it. If it's important logic to keep around, I'd probably put it in a branch.